cw2dmk.c

```diff
@@ -1802,6 +1802,7 @@ main(int argc, char** argv)
   int cw_mk = 1;
   char *replay = NULL;
   FILE *replay_file = NULL;
+  char optname[3] = "-?";
```

Comment:
Another way to have done this is just before calling `getopt()` save `optind` in a var (e.g. `ch_optind`) then instead of calling `strto.*strict()` with `optname`, use `argv[ch_optind]` (and at the bottom of the `while` loop, `ch_optind = optind;` to update).

Reply:
Does that work? getopt handles option arguments that are either part of the same argv[] entry or in the next one, such as -v1 versus -v 1. So if a user wrote -v1z, wouldn't the optind method complain that "-v1z requires a numeric argument" instead of "-v requires a numeric argument"? Even if it works for some reason, it seems more confusing.

Reply:
Oh, yes.  Silly oversight on my part not considering combined options without option-arguments.